src/main/java/seedu/command/ModificationCommand.java

```diff
@@ -15,10 +17,12 @@
     public static final String INVALID_TYPE_MESSAGE = "Equipment Type needs to be one of:" + System.lineSeparator()
             + EquipmentType.getAllTypes();
     public static final String INVALID_COST_MESSAGE = "Cost needs to be able to be parsed to a double";
+    public static final String INVALID_DATE_MESSAGE = "Date needs to be able to be parsed, make sure you're using "
```

Comment:
Oh and, I think maybe we need to change CheckCommand as well? Because there is Pair being used there as well and the date is being passed to EquipmentManager through CheckCommand also

Reply:
Perhaps all the invalid messages can be moved to Command?